test/second_q/mappers/test_bosonic_logarithmic_mapper.py

```diff
@@ -0,0 +1,522 @@
+            2,
+            2j,
+            2j,
+            -2,
+            -2j,
+            2,
+            2,
+            2j,
+        ],
+    )
+
+    bos_op4 = BosonicOp({"-_1": 1})
+    # Using: max_occupation = 3 (number_of_qubits_per_mode = 2)
+    ref_qubit_op4_nq2 = 0.25 * SparsePauliOp(
+        ["IXII", "IYII", "ZXII", "ZYII", "XXII", "XYII", "YXII", "YYII"],
+        coeffs=[
+            1 + sq_3,
+            1j * (1 + sq_3),
+            1 - sq_3,
+            1j * (1 - sq_3),
+            sq_2,
+            -1j * sq_2,
+            1j * sq_2,
+            sq_2,
+        ],
+    )
+    # Using: max_occupation = 7 (number_of_qubits_per_mode = 3)
+    ref_qubit_op4_nq3 = 0.125 * SparsePauliOp(
+        [
+            "IIXIII",
+            "IIYIII",
+            "IZXIII",
+            "IZYIII",
+            "ZIXIII",
+            "ZIYIII",
+            "ZZXIII",
+            "ZZYIII",
+            "IXXIII",
+            "IXYIII",
+            "IYXIII",
+            "IYYIII",
+            "ZXXIII",
+            "ZXYIII",
+            "ZYXIII",
+            "ZYYIII",
+            "XXXIII",
+            "XXYIII",
+            "XYXIII",
+            "XYYIII",
+            "YXXIII",
+            "YXYIII",
+            "YYXIII",
+            "YYYIII",
+        ],
+        coeffs=[
+            1 + sq_3 + sq_5 + sq_7,
+            1j * (1 + sq_3 + sq_5 + sq_7),
+            1 - sq_3 + sq_5 - sq_7,
+            1j * (1 - sq_3 + sq_5 - sq_7),
+            1 + sq_3 - sq_5 - sq_7,
+            1j * (1 + sq_3 - sq_5 - sq_7),
+            1 - sq_3 - sq_5 + sq_7,
+            1j * (1 - sq_3 - sq_5 + sq_7),
+            sq_2 + sq_6,
+            -1j * (sq_2 + sq_6),
+            1j * (sq_2 + sq_6),
+            sq_2 + sq_6,
+            sq_2 - sq_6,
+            -1j * (sq_2 - sq_6),
+            1j * (sq_2 - sq_6),
+            sq_2 - sq_6,
+            2,
+            -2j,
+            -2j,
+            -2,
+            2j,
+            2,
+            2,
+            -2j,
+        ],
+    )
+
+    bos_op5 = BosonicOp({"+_0 -_0": 1})
+    # Using: max_occupation = 3 (number_of_qubits_per_mode = 2)
+    ref_qubit_op5_nq2 = 0.5 * SparsePauliOp(["II", "IZ", "ZI"], coeffs=[3, -1, -2])
+    # Using: max_occupation = 2
+    ref_qubit_op6_nq3 = SparsePauliOp(
+        ["III", "IZZ", "IZI", "IIZ", "ZZI", "ZII"], coeffs=[0.75, -0.25, 0.25, 0.25, -0.5, -0.5]
+    )
+
+    bos_op6 = BosonicOp({"-_0 +_0": 1})
+    # Using: max_occupation = 3 (number_of_qubits_per_mode = 2)
+    ref_qubit_op6_nq2 = 0.5 * SparsePauliOp(["II", "IZ", "ZZ"], coeffs=[3, 1, -2])
+    # Using: max_occupation = 2
+    ref_qubit_op6_tr2 = SparsePauliOp(
+        ["III", "IZZ", "IZI", "IIZ", "ZZI", "ZII"], coeffs=[0.75, -0.25, 0.25, 0.25, -0.5, -0.5]
+    )
+
+    bos_op7 = BosonicOp({"+_0 -_1": 1})
+    bos_op8 = BosonicOp({"-_1 +_0": 1})
+    # Using: max_occupation = 3 (number_of_qubits_per_mode = 2)
+    ref_qubit_op7_8_nq2 = 0.0625 * SparsePauliOp(
+        [
```

Comment:
I suggest you either extract these expected outcomes into some file which you load, or you add `# fmt: off` and `# fmt: on` comments around this and manually format it in a more readable manner.
Same in the other places where these lists get very long.